src/core/libmaven/mavenparameters.cpp

```diff
@@ -309,6 +314,10 @@ void MavenParameters::copyFrom(const MavenParameters& mp)
     // vectors are meant to be used only by the global `MavenParameters` object,
     // which will not be created by a copy operation like this.
     samples = mp.samples;
+
+    peakMlModelType = mp.peakMlModelType;
+    badGroupUpperLimit = mp.badGroupUpperLimit;
+    goodGroupLowerLimit = mp.goodGroupLowerLimit;
```

Comment:
Why not also copy the `MavenParameters::classifyUsingPeakMl` attribute? Makes sense to do it and you can use it to selectively show or hide PeakML attributes in "setting log" widget.